cmd/lookoutd/common.go

```diff
@@ -113,6 +113,23 @@ func (c *lookoutdCommand) initConfig() (Config, error) {
 		return conf, fmt.Errorf("Can't parse configuration file: %s", err)
 	}
 
+	// Set default timeouts
```

Comment:
I would suggest to rewrite it like this:
```
conf.Timeout = TimeoutConfig{
    AnalyzerReview: 10 * time.Minute,
    AnalyzerPush: 60 * time.Minute,
    ...
}

if err := yaml.Unmarshal([]byte(configData), &conf); err != nil {
```

Reply:
:+1: done in 9b3c61f